resources/clients/react/elros-pam/README.md

```diff
@@ -22,7 +22,7 @@ integrates with Amazon Web Services.
 
 ## Add new components
 
-Once you have a copy of the boilerplate code you can treat it like
+When you have a copy of the boilerplate code you can treat it like
 any other React application. Add new components, change the folder
 structure, make it yours.
```

Comment:
```suggestion
Feel free to update or add new React components to broaden or refine this use case.
```

Reply:
Disagree. This README is aimed both at our team and customers. Both parties should feel comfortable making changes without worrying about breaking something.